### PR TITLE
add id as primary key for independent practice

### PR DESCRIPTION
**Pull request by aedward8**
Since we have been learning the importance of including a primary key in our table I updated the independent practice to include a primary key column

### CODE REVIEW

File: databases/sql-1.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Most software applications store data of some sort -- user data, content, etc. D
 
 1. PostgreSQL is a popular, open-source version of SQL. Try it out:
    - In Terminal, type `psql` to use PostgreSQL.
-   - Try adding a table with the command `CREATE TABLE food (name TEXT, calories INTEGER);`
+   - Try adding a table with the command `CREATE TABLE food (id INTEGER,name TEXT, calories INTEGER);`
```
Comment (lalpert) [inline databases/sql-1.md:60]: Could you include a space after the comma here to be consistent?

**Review by lalpert — COMMENTED** [review]
Good idea! I commented on one small change to make. And there's a git conflict so make sure to resolve that.

File: databases/sql-1.md (hunk 1 above)
Comment (aedward8) [reply databases/sql-1.md]: Hi @lalpert  thank you for your comment I added the comma and spacing as well as fixed the conflict which was that I added `\dt` to display table.

File: databases/sql-1.md [hunk 2]
```diff
@@ -55,21 +55,21 @@ Most software applications store data of some sort -- user data, content, etc. D
...
 1. Work through the [Codecademy SQL Tutorial](https://www.codecademy.com/learn/learn-sql) **Section 1 (Manipulation)**:
 
-1. PostgreSQL is a popular, open-source version of SQL. Try it out:
+2. PostgreSQL is a popular, open-source version of SQL. Try it out:
    - In Terminal, type `psql` to use PostgreSQL.
-   - Try adding a table with the command `CREATE TABLE food (name TEXT, calories INTEGER);`
-   - See that the table was created by typing `\dt` to see the list of all existing tables.
-   - Try adding some data: `INSERT INTO food VALUES ('pizza', 500);`
+   - Try adding a table with the command `CREATE TABLE food (id INTEGER, name TEXT, calories INTEGER);`
+   - See that the table was created by typing `.\dt` to see the list of all existing tables.
```
Comment (lalpert) [inline databases/sql-1.md:61]: There shouldn't be a `.` before `\dt`

File: databases/sql-1.md [hunk 3]
```diff
@@ -55,21 +55,21 @@ Most software applications store data of some sort -- user data, content, etc. D
...
+   - Try adding some data: `INSERT INTO food VALUES ("pizza", 500);`
```
Comment (lalpert) [inline databases/sql-1.md:62]: double-quotes aren't the same as single-quotes in SQL so this needs to be changed back to single quotes.

Also, now that there is an "id" column in the food table, this insert statement should include a value for id as well.

You should be able to double-check that your changes are correct by running the new commands in SQL and seeing that they work -- let me know if you hit any problems doing that!

File: databases/sql-1.md [hunk 4]
```diff
@@ -55,21 +55,21 @@ Most software applications store data of some sort -- user data, content, etc. D
 
 1. Work through the [Codecademy SQL Tutorial](https://www.codecademy.com/learn/learn-sql) **Section 1 (Manipulation)**:
 
-1. PostgreSQL is a popular, open-source version of SQL. Try it out:
+2. PostgreSQL is a popular, open-source version of SQL. Try it out:
```
Comment (lalpert) [inline databases/sql-1.md:58]: It's fine to leave in the re-numbered list items, but just fyi, markdown automatically puts in the right list numbers for you when rendering the page. People often use all 1's in markdown lists to make it easier to add intermediate steps later without needing to renumber the list.

**Review by lalpert — COMMENTED** [review]
Thanks for making those changes! I see some other changes in this PR too now so I added comments on those.

File: databases/sql-1.md (hunk 4 above)
Comment (aedward8) [reply databases/sql-1.md:58]: @alodahl  I believe this issue is resolved and ready for merge

**Pull request merged**